lib/git-manager.js

```diff
@@ -72,19 +72,21 @@ module.exports = {
       // before errors happen.
       githubPull.body = githubPull.body || '';
 
+      var reviewComments = module.exports.getPullReviewComments(githubPull.number);
       var comments = module.exports.getPullComments(githubPull.number);
       var headCommit = module.exports.getCommit(githubPull.head.sha);
       var commitStatus = module.exports.getCommitStatus(githubPull.head.sha);
       var events = module.exports.getPullEvents(githubPull.number);
 
       // Returned to the map function. Each element of githubPulls maps to
       // a promise that resolves to a Pull.
-      return Promise.all([comments, headCommit, commitStatus, events])
+      return Promise.all([reviewComments, comments, headCommit, commitStatus, events])
        .then(function(results) {
-         var comments = results[0],
-             headCommit = results[1],
-             commitStatus = results[2],
-             events = results[3];
+         var reviewComments = results[0],
+             comments       = results[1],
+             headCommit     = results[2],
+             commitStatus   = results[3],
+             events         = results[4];
```

Comment:
I think this will be fine, since the results array was only used for binding promise results to local variables. The return value from `parse` is a new `Pull` object, so nothing should be using `results` after this function returns.